### ISSUE TITLE
Unbroken review string is causing layout issues in Themes Reviews Logs and "Your review history" pages

### ISSUE THREAD

**Issue by ValentinaPC** [mozilla/addons-server#6769]
Steps to reproduce:
1. Submit a theme i.e. https://addons.allizom.org/en-US/developers/theme/strings123/edit
2. Add an unbroken string while flagging the submitted theme 
3. Go to Reviewer Tools - Themes - Logs - Reviews https://addons.allizom.org/en-US/editors/themes/logs
4. Click on the "show comments" for the above theme

Expected results:
Comment is wrapped.

Actual results:
Page layout is broken.

Notes/Issues:
- "Your review history" page is also broken - https://addons.allizom.org/en-US/editors/themes/history/ from the same reason.
Verified on AMO-stage FF56(Win 7). Reproducing in AMO-dev also.
Video for this issue:
![unbr](https://user-images.githubusercontent.com/15685960/32004625-e753bd00-b9aa-11e7-817f-5c4da6225919.gif)

**Comment by jvillalobos** [comment]
Mentor: @eviljeff

**Comment by VishalCR7** [comment]
@eviljeff  I would like to take this up.

**Comment by VishalCR7** [comment]
@eviljeff  can you please review the PR #7382

**Comment by ioanarusiczki** [comment]
I verified this issue on AMO dev using FF58 (Win10).
I verified Reviewer Tools - Themes - Logs - Reviews and  "Your review history" page.  Comments are wrapped when an unbroken string is added while flagging the submitted theme.

![text1](https://user-images.githubusercontent.com/33448286/35384553-4c54573a-01ce-11e8-80ac-9a9aadda336c.png)

![text2](https://user-images.githubusercontent.com/33448286/35384559-50553188-01ce-11e8-9513-fd89b15ea46c.png)
